TF_Service_dotNet/TouchFreeTests/PositioningModuleTests.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using Moq;
+using NUnit.Framework;
+using TouchFreeTests.TestImplementations;
+using Ultraleap.TouchFree.Library;
+using Ultraleap.TouchFree.Library.Interactions;
+using Ultraleap.TouchFree.Library.Interactions.PositionTrackers;
+
+namespace TouchFreeTests
+{
+    class PositioningModuleTests
+    {
+        TestVirtualScreenManager virtualScreenManager = new TestVirtualScreenManager();
+
+        private PositioningModule CreatePositioningModule(IEnumerable<IPositionTracker> positionTrackers)
+        {
+            if (!positionTrackers.Any(x => x.TrackedPosition == TrackedPosition.INDEX_STABLE))
+            {
+                positionTrackers = positionTrackers.Append(new IndexStableTracker());
+            }
+
+            Mock<IPositionStabiliser> stabiliser = new Mock<IPositionStabiliser>();
+            stabiliser.Setup(x => x.ApplyDeadzone(It.IsAny<Vector2>())).Returns<Vector2>(v => v);
+
+            PositioningModule positioningModule = new PositioningModule(stabiliser.Object, virtualScreenManager,
+                positionTrackers);
+            positioningModule.TrackedPosition = TrackedPosition.INDEX_TIP;
+
+            return positioningModule;
+        }
+
+        private IPositionTracker CreateTrackerWithPosition(TrackedPosition trackedPosition, float x, float y, float z)
+        {
+            Mock<IPositionTracker> mockTracker = new Mock<IPositionTracker>();
+            mockTracker.SetupGet(x => x.TrackedPosition).Returns(trackedPosition);
+            mockTracker.Setup(x => x.GetTrackedPosition(It.IsAny<Leap.Hand>())).Returns(new Vector3(x, y, z));
+            return mockTracker.Object;
+        }
+
+        [TestCase(0, 0.5f, 3, 100, 200, 1, 50, 100, 3)]
```

Comment:
Same comment as previous here, if we can wrap these in more complex types (or even just arrays) for readability, I'd prefer that 👍